ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -738,6 +738,17 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-vsphere-upi-disconnected-sdn-workers-rhcos-rhel8
+- as: aws-ipi-hypershift-install-test
```

Comment:
@LiangquanLi930 please follow https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/README.md to name your test name. 
it is required by qe-productivity team.

and for `cron`, I suppose your test is p2, so it should be `cron: 0 3 */7 * *`

Reply:
@kuiwang02 I see managed HyperShift SLO is mainly about install & uninstall process. To save resources of prow, could we run it every 2 days ... ?

Reply:
@heliubj18 I can not answer you. you should query it with qe-productivity team.

Note: P2 means run it every 7 days.

Reply:
@liangxia We just want to trigger this test 2 times a week, how could we use cron and level? could you PTAL when you 
are available? thanks

Reply:
You can use p1 (daily) or p2 (weekly) currently.   There is a [PR](https://github.com/openshift/release/pull/32303) ongoing which allow more levels of test frequency.

Reply:
so, I want use `f3`, i need to wait for this [PR](https://github.com/openshift/release/pull/32303) merge?

Reply:
There is no guarantee when that PR will merge.
So according to rules we currently have, you can use either p1 (daily), or p2 (weekly)
We can update later if that PR got merge.